example/android/settings.gradle

```diff
@@ -13,3 +13,9 @@ plugins.each { name, path ->
     include ":$name"
     project(":$name").projectDir = pluginDirectory
 }
+
+// Run enableLocalBuild task to enable building purchases-hybrid-common from your local copy
+if (file(".composite-enable").exists()) {
+    includeBuild ('../../../../purchases-hybrid-common/android')
```

Comment:
is this line assuming the local path of purchases-hybrid-common?

Reply:
yes, what could we do to make this easier? maybe an environment variable?

Reply:
env variable would be the absolute ideal

Reply:
let me try that